pkgs/applications/editors/vscode/extensions/ms-vscode.cpptools/default.nix

```diff
@@ -30,14 +30,20 @@
 
 let
   gdbDefaultsTo = if gdbUseFixed then "${gdb}/bin/gdb" else "gdb";
+  base = if stdenv.hostPlatform.system == "x86_64-linux" then {
```

Comment:
Why use `stdenv.hostPlatform.system` instead of `stdenv.system`?

Also, just a suggestion if you want to eliminate the if clauses, you can use a struct expression similar to this: https://github.com/NixOS/nixpkgs/pull/255185/files#diff-1e1071c682940fc6a95a879445b64519dae98cb30e354e9ea191f119259780ccR15

Reply:
No reason. I saw `stdenv.hostPlatform.system` somewhere and continued using it.

Reply:
@StarGate01 This is fixed now